example/test_driver/flutter_mailer_e2e_test.dart

```diff
@@ -0,0 +1,11 @@
+import 'dart:io';
+
+import 'package:flutter_driver/flutter_driver.dart';
```

Comment:
This can be replaced with something like this: 

```
import 'dart:async';

import 'package:e2e/e2e_driver.dart' as e2e;

Future<void> main() async => e2e.main();
```

Which then should link it up properly with the device tests (so when you run device tests using ./gradlew for Android), e2e will run your tests.